rudder-web/src/main/webapp/javascript/rudder/homePage.js

```diff
@@ -175,6 +178,9 @@ function displayInventoryGraph (id,data) {
            }
          }
        }
+      , color: {
+        pattern: ['rgb(31, 119, 180)', 'rgb(23, 190, 207)', 'rgb(255, 127, 14)', 'rgb(255, 224, 14)', 'rgb(227, 119, 194)', 'rgb(44, 160, 44)', 'rgb(255, 104, 105)', 'rgb(148, 103, 189)', 'rgb(140, 86, 75)', 'rgb(160, 160, 160)', 'rgb(146, 201, 74)', '#ffd203', 'rgb(132, 63, 152)']
```

Comment:
and use the variable here, so when you will have to modify the colors, you will have to modify only at one place :)